app/containers/message/Components/CollapsibleQuote.tsx

```diff
@@ -0,0 +1,183 @@
+import { transparentize } from 'color2k';
+import { dequal } from 'dequal';
+import React, { useContext, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { themes } from '../../../constants/colors';
+import { IAttachment } from '../../../definitions/IAttachment';
+import { TGetCustomEmoji } from '../../../definitions/IEmoji';
+import { CustomIcon } from '../../../lib/Icons';
+import sharedStyles from '../../../views/Styles';
+import Markdown from '../../markdown';
+import MessageContext from '../Context';
+import Touchable from '../Touchable';
+
+const styles = StyleSheet.create({
+	button: {
```

Comment:
I got `height: 35.3` on iOS.
Maybe we should set a minimum height to 40 and vertically center?